package.json

```diff
@@ -100,7 +100,8 @@
     "prebuild": "rimraf dist/ && rimraf build/ && mkdirp dist && mkdirp build",
     "build": "tsc --module commonjs && rollup -c rollup.config.ts && npm run build:create-reftest-list && npm run build:testrunner && npm run build:minify",
     "build:testrunner": "rollup -c tests/rollup.config.ts",
-    "build:minify": "uglifyjs --compress --comments /^!/ -o dist/html2canvas.min.js --mangle -- dist/html2canvas.js",
+    "build:minify": "uglifyjs --compress --comments /^!/ -o dist/html2canvas.min.js --mangle -- dist/html2canvas.js && npm run build:minify:esm",
+    "build:minify:esm": "uglifyjs --compress --comments /^!/ -o dist/html2canvas.esm.min.js --mangle -- dist/html2canvas.esm.js",
```

Comment:
Why `esm` built was added?

Reply:
I'm interested because it makes a build process longer because build:minify executes now build:minify:esm, but only `html2canvas.min.js` is used.

Reply:
`esm` could be potentially used in future with dynamic import, minified version may be useful for that.
And `esm` was already part of build process as well as some builds for tests. Only minification was added here.

Reply:
By the way, what is a difference between build:minify and build:minify:esm? They are the same, aren't they?

Reply:
One minifies `html2canvas.js`, another - `html2canvas.min.js`

Reply:
Right, thanks. But my main confusion is why do we need to minify both versions of the library but use only one in Surfly?

Reply:
Just to be clear, I don't mind of having another rule `build:minify:esm`, I just don't understand why it should be executed together with  `build:minify`.

Reply:
As I mentioned before: it would be good to switch from global scoped library to esm import in surfly. For that we'll need minified esm lib.
From the other side adding such minification later (once it's actually needed) will require lot of extra steps to create similar PR. So it's easier to do it all at once now.